spec/datadog/ci/ext/environment_spec.rb

```diff
@@ -30,7 +30,7 @@
         context "for fixture #{File.basename(filename)}" do
           # Create a context for each example inside the JSON fixture file
           JSON.parse(f.read).each_with_index do |(env, tags), i|
-            context "##{i}" do
+            context i.to_s do
```

Comment:
Rubocop was complaining of [RSpec/ContextMethod](https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/ContextMethod) cop.

Since `i` doe snot represent a method name, I decided to drop the `#` sign